CSS_files/components/alerts/alerts.css

```diff
@@ -0,0 +1,26 @@
+.alert{
+    padding: 1.6rem 2.4rem;
+    font-size: 1.4rem;
+    border-radius: 6px;
+    text-align: left;
+    width: 100%;
+}
+
+.alert-primary{
+    background: #93C5FD;
+    color: #2563EB;
+}
```

Comment:
It will be good if you define these color variables in one file

Reply:
Yes , I directly used the hex codes here because I don't need these color anywhere else in my project.